modules/logging-loki-gui-install.adoc

```diff
@@ -0,0 +1,87 @@
+
+{Product-title} Operators use custom resources (CR) to manage applications and their components. High-level configuration and settings are provided by the user within a CR. The Operator translates high-level directives into low-level actions, based on best practices embedded within the Operator’s logic. A custom resource definition (CRD) defines a CR and lists all the configurations available to users of the Operator. Installing an Operator creates the CRDs, which are then used to generate CRs.
+
+.Prerequisites
+
+* Supported object store (AWS S3, Google Cloud Storage, Azure, Swift, Minio, OpenShift Data Foundation)
+
+.Procedure
+
+. Install the `Loki Operator`:
+
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+.. Type `Loki Operator` in the filter by keyword box. Choose *Loki Operator* from the list of available Operators and click *Install*.
++
+[NOTE]
+====
+The Community Loki Operator is not supported by Red Hat.
+====
+
+.. On the Install Operator page, for *Update Channel* select *stable*.
++
+--
+include::snippets/logging-stable-updates-snip.adoc[]
+--
+
+As the Loki Operator must be deployed to the global operator group namespace `openshift-operators-redhat`, *Installation mode* and *Installed Namespace* is already be selected. If this namespace does not already exist, it is created for you.
+
+.. Select *Enable operator-recommended cluster monitoring on this namespace.*
+
+This option sets the `openshift.io/cluster-monitoring: "true"` label in the Namespace object. You must select this option to ensure that cluster monitoring scrapes the `openshift-operators-redhat` namespace.
+
+.. For *Update approval* select *Automatic*, then click *Install*.
++
+--
+include::snippets/logging-approval-strategy-snip.adoc[]
+--
+
+.Procedure
+
+. Install the `Red Hat OpenShift Logging` Operator:
+
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+.. Type `OpenShift Logging` in the filter by keyword box. Choose *Red Hat OpenShift Logging* from the list of available Operators and click *Install*.
+
+.. On the Install Operator page, under *Update channel* select *stable*.
++
+--
+include::snippets/logging-stable-updates-snip.adoc[]
+--
+
+As the `Red Hat OpenShift Logging` Operator is only deployable to the `openshift-logging` namespace, *Installation mode* and *Installed Namespace* is already selected. If this namespace does not already exist, it is created for you.
+
+.. If you are creating the `openshift-logging` namespace, select the option to *Enable Operator recommended cluster monitoring on this Namespace*.
++
+[NOTE]
+====
+If the `openshift-logging` namespace already exists, you must add the namespace label, `openshift.io/cluster-monitoring: "true"`, to enable metrics service discovery.
+====
+
+.. Under *Update approval* select *Automatic*.
++
+--
+include::snippets/logging-approval-strategy-snip.adoc[]
+--
```

Comment:
If you're using snippets, the metadata for the snippets must be updated, i.e. the list of places where it's used in the snippet file itself: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#text-snippet-file-metadata

Please make sure that metadata is updated for all modules and snippets